Sources/UIScreen+render.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  UIScreen+render.swift
+//  UIKit
+//
+//  Created by Chris on 08.08.17.
+//  Copyright © 2017 flowkey. All rights reserved.
+//
+
+import SDL
+import SDL_gpu
+import func Foundation.round
+
+extension UIScreen {
+    func render(window: UIWindow?, atTime frameTimer: Timer) {
```

Comment:
I asked myself if it makes sense to pass window as optional. Maybe we unwrap it in UIApplication and call render only when window exists.

Reply:
Yes I'm not 100% sure on this yet either.

The main thing is that `UIApplication.keyWindow` is optional itself, so it made more sense to me to just pass that variable in, rather than checking every time from the outside. The API's intention is that it can *try* to render the UIApplication's keyWindow (or another if you want) but it shouldn't break otherwise

Reply:
ok, fair enough